Makefile

```diff
@@ -10,7 +10,7 @@ run:
 	cd $(APP_NAME) && \
 	docker-compose -f docker-compose.test.yml run test
 
-base_spec = spec/base/**/*_spec.rb,spec/addons/docker/**/*_spec.rb
+base_spec = spec/base/**/*_spec.rb,spec/addons/docker/**/*_spec.rb,spec/addons/terraform/**/*_spec.rb
```

Comment:
The Makefile was refactored in #248, so please rebase this PR @namtx

Reply:
rebased and made some changes already, thanks for pointing